src/foam/u2/wizard/StepWizardController.js

```diff
@@ -414,14 +414,14 @@ foam.CLASS({
         if ( ! nextWizardlet ) break;
 
         // Load the next available wizardlets; ignore unavailable wizardlets
-        while ( ! nextWizardlet.isAvailable ) {
+        while ( nextWizardlet && ! nextWizardlet.isAvailable ) {
           debugLog('skipping unavailable wizardlet', nextPosition, {
             wizardlet: nextWizardlet
           });
           nextPosition = nextPosition.getNext(this.wizardlets);
           nextWizardlet = nextPosition && this.wizardlets[nextPosition.wizardletIndex];
-          if ( ! nextWizardlet ) break;
         }
+        if ( ! nextWizardlet ) break;
```

Comment:
Just checking this break is for       
while ( referencePosition ) {
looked like an easy copy and paste mistake.

Reply:
it's intentional for it to break out of the loop since the nextWizardlet is null.